src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -351,6 +351,15 @@ namespace topology
                                                   conduit::Node &s2dmap,
                                                   conduit::Node &d2smap);
 
+        //---------------------------------------------------------------------
```

Comment:
add a note here (the header) that this variant of the function call will also map the fields 
also add the details about what can be passed via the options node.

Reply:
ok